client/src/d_netinfo.cpp

```diff
@@ -110,6 +110,13 @@ team_t D_TeamByName (const char *team)
 			return (team_t)i;
 	}
 
+	if (stricmp(team, "0") == 0)
+		return TEAM_BLUE;
+	else if (stricmp(team, "1") == 0)
+		return TEAM_RED;
+	else if (stricmp(team, "2") == 0)
+		return TEAM_GREEN;
+
```

Comment:
You can get away with a plain old `strcmp` here - case-sensitivity doesn't matter for numbers.